src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -1,20 +1,22 @@
 import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
-import { useIntl } from 'react-intl';
+import { FormattedMessage, useIntl } from 'react-intl';
 import HeaderSection from '../shared/HeaderSection/HeaderSection';
 import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
 import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
 import { Switch } from '../shared/Switch/Switch';
 import { FieldGroup, FieldItem, NumberField, SubmitButton } from '../form-helpers/form-helpers';
 import { Form, Formik } from 'formik';
 import useTimeout from '../../hooks/useTimeout';
+import { InjectAppServices } from '../../services/pure-di';
 
-export const ContactPolicy = () => {
-  const [active, toggleActive] = useState(false);
-  const [formSubmitted, setFormSubmitted] = useState(false);
+export const ContactPolicy = InjectAppServices(({ dependencies: { experimentalFeatures } }) => {
+  const [isActive, setIsActive] = useState(false);
+  const [isFormSubmitted, setIsFormSubmitted] = useState(false);
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
-
+  const isContactPolicyEnabled =
```

Comment:
When the api for features is ready we should store all features in session for example and then get this information from there.

Reply:
Sure! This is just a temporary solution.